tests/capture/workflow/test_capture_qnode.py

```diff
@@ -1881,6 +1883,7 @@ def circuit(x):
         assert spy.call_count > 1
 
     # pylint: disable=unused-argument
+    @pytest.mark.xfail  # think v0.5.3 broke dynamic shapes and static argnums
```

Comment:
```suggestion
    @pytest.mark.xfail  # think JAX 0.5.3 broke dynamic shapes and static argnums
```

for future reference : )